plugins/flytekit-k8s-pod/flytekitplugins/pod/task.py

```diff
@@ -113,6 +113,9 @@ def get_k8s_pod(self, settings: SerializationSettings) -> _task_models.K8sPod:
             ),
         )
 
+    def get_sql(self, settings: SerializationSettings) -> _task_models.Sql:
```

Comment:
actually why does this need to be overridden?  shouldn't this inherit from the base class?

Reply:
sorry, yup. we can remove it.